benchmarks/circuit_construction.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2022 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Set
+
+import cirq
+
+
+def get_qubits(distance: int):
```

Comment:
Alternatively (not sure which is preferable):
```python
data_qubits = {GridQubit(2*x+1, 2*y+1) for x in range(d) for y in range(d)}
z_measure_qubits = {GridQubit(2*x, 2*y) for x in range(1, d) for y in range(d+1) if x % 2 == y % 2}
x_measure_qubits = {GridQubit(2*x, 2*y) for x in range(d+1) for y in range(1, d) if x % 2 != y % 2}
```

Reply:
changed to the suggestion.